docs/detections/detections-index.asciidoc

```diff
@@ -2,6 +2,8 @@ include::detection-engine-intro.asciidoc[]
 
 include::about-rules.asciidoc[]
 
+include::detections-req.asciidoc[leveloffset=+1]
```

Comment:
I'm debating whether we should swap this page with the "About detection rules" page. The "Detections prerequisites and requirements" is about privs needed to use rules _and alerts_, so it might be more appropriate to place it closer to the "Detections and alerts" page, instead of in between rule-related content.

Reply:
I'm open to that! Other reviewers?

Reply:
I agree, that makes sense!

Reply:
Agree!

Reply:
I agree with this.